libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -101,7 +101,7 @@ export class SyslumennService {
 
   async sealCriminalRecord(criminalRecord: string): Promise<SvarSkeyti> {
     const { id, api } = await this.createApi()
-    const explination = 'Undirritað af sýslumanni'
+    const explination = 'Rafrænt undirritað vottorð'
```

Comment:
explination - explanation - minor typo that doesn't impact anything.

However, perhaps the string shouldn't be hardcoded?